tests/generator-Tests/SupportFiles/DebuggerDisableUserUnhandledExceptionsAttributeAttribute.cs

```diff
@@ -0,0 +1,13 @@
+#if !NET9_0_OR_GREATER
+
+using System;
+
+namespace System.Diagnostics
+{
+	// This attribute was added in .NET 9, and we may not be targeting .NET 9 yet.
+	public class DebuggerDisableUserUnhandledExceptionsAttributeAttribute : Attribute
```

Comment:
Not sure if deliberate or accidental, but the .NET type has only *one* "Attribute" suffix, not two: [`DebuggerDisableUserUnhandledExceptionsAttribute`](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.debuggerdisableuserunhandledexceptionsattribute?view=net-9.0)

Reply:
Doh, surprisingly this works because we were generating `[DebuggerDisableUserUnhandledExceptionsAttribute]` and it removes one "Attribute".

Updated the generated code to `[DebuggerDisableUserUnhandledExceptions]` to match (most) of the other attributes we generate, and fixed this support file.